client/question/services/code_evaluators/PythonCodeRunnerServiceSpec.js

```diff
@@ -118,6 +118,10 @@ describe('PythonCodeRunnerService', function() {
       [false, false, false]];
     responseDict.results[VARNAME_PERFORMANCE_TEST_RESULTS] = [
       ['linear']];
+    // eslint-disable-next-line camelcase
+    responseDict.memory_limit_exceeded = false;
```

Comment:
Ah! I think, if you did

    responseDict = {
      stdout: ...,
      stderr: ...,
      ...
    }

it _might_ work fine, not sure. But what you have here is fine too IMO.